YOLayout/AppKit/YONSView.h

```diff
@@ -60,3 +63,7 @@
 - (void)layoutView;
 
 @end
+
+// Deprecated
+@interface YONSView : YOView
```

Comment:
We could probably just blow it away, I doubt at this point there are many people using this on OSX.